src/os/external/read-env-variable.md

```diff
@@ -0,0 +1,25 @@
+## Read Environment Variable 
+
+[![std-badge]][std] [![cat-os-badge]][cat-os]
+
+Reads an environment variable via [std::env::var].
+
+```rust,edition2018,no_run
+use std::env;
+use std::fs;
+use std::io::Error;
+
+fn main() -> Result<(), Error> {
+    // read `config_path` from the environment variable `CONFIG`.
+    // If `CONFIG` isn't set, fall back to a default config path.
+    let config_path = env::var("CONFIG")
+        .unwrap_or("<default config path>".to_string());
```

Comment:
love the unwrap_or usage here, but lets give it a static path, like /etc/config so at a glance it is obvious how to repurpose the recipe.

Reply:
Sure thing :wink: